app/bundles/PluginBundle/Helper/IntegrationHelper.php

```diff
@@ -240,21 +250,24 @@ public function getIntegrationObjects($specificIntegrations = null, $withFeature
             }
 
             if (!isset($integrations[$integrationName])) {
-                $integration     = $available[$integrationName];
```

Comment:
I need to add back the manual class name building & instantiation to keep BC for 3rd party plugins, at least until 3.0. Here, line 105 and line 166